app/ui/src/app/common/loading/loading.component.html

```diff
@@ -1,6 +1,11 @@
-<ng-container *ngIf="loading">
+<ng-container *ngIf="loading; else body">
```

Comment:
I think you can get away with just 

```
<ng-container *ngIf="loading; else content">
  <div class="syn-loading">
      <div class="spinner spinner-lg loading"></div>
  </div>
</ng-container>
```

Reply:
I think I could but I kinda wanted to keep support for *not* using an *ng-template* still also.

Reply:
Oh, I didn't think about that, you're right!